### PR TITLE
[RHACS] Updated release notes for patch release

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Updated release notes for the patch release

Preview: 
- https://openshift-docs-git-651-patch-gnelson.vercel.app/openshift-acs/master/release_notes/365-release-notes.html
- https://openshift-docs-git-651-patch-gnelson.vercel.app/openshift-acs/master/release_notes/364-release-notes.html

NOTES:
- This PR is for RHACS docs at https://docs.openshift.com/acs/welcome/index.html
- It doesn't require any special labels or milestones

### CODE REVIEW

File: release_notes/364-release-notes.adoc [hunk 1]
```diff
@@ -33,6 +33,15 @@ The bug caused internal communication failure, and the RHACS services would fail
...
 * *ROX-7872*: The updated Operator image sets the memory limit to 1 GiB and memory requests to 200 MiB to address out-of-memory issues when using the RHACS Operator at scale.
 
+[id="resolved-in-version-3642"]
+=== Resolved in version 3.64.2
+
+Release date: September 21, 2021
+
+* *ROX-8008*: Previously, you could not use URN-based IdP Issuers while configuring SAML identity providers. This has been fixed.
+* *ROX-8033*: Due to how {product-title} previously addressed its internal service endpoints, OpenShift clusters with enabled proxy failed to download the correct kernel probes.
+* *ROX-8034*: Previously, if you were using backported 5.11 kernels for Ubuntu 20.04, sometimes, Collector failed on upgrade due to a change in the Ubuntu kernel build.
```
Comment (skrthomas) [inline release_notes/364-release-notes.adoc:43]:
```suggestion
* *ROX-8034*: Previously, if you were using backported 5.11 kernels for Ubuntu 20.04, the Collector sometimes failed on upgrade due to a change in the Ubuntu kernel build.
```

File: release_notes/365-release-notes.adoc [hunk 2]
```diff
@@ -40,6 +40,15 @@ See xref:../installing/install-ocp-operator.adoc#addmission-controller-settings_
...
 * *ROX-7861*: Previously, {product-title} compliance control NIST 800-190 Control 4.1.4 did not correctly detect policies used for secrets protection.
 
+[id="resolved-in-version-3651"]
+=== Resolved in version 3.65.1
+
+Release date: September 21, 2021
+
+* *ROX-8008*: Previously, you could not use URN-based IdP Issuers while configuring SAML identity providers. This has been fixed.
+* *ROX-8033*: Due to how {product-title} previously addressed its internal service endpoints, OpenShift clusters with enabled proxy failed to download the correct kernel probes.
+* *ROX-8034*: Previously, if you were using backported 5.11 kernels for Ubuntu 20.04, sometimes, Collector failed on upgrade due to a change in the Ubuntu kernel build.
```
Comment (skrthomas) [inline release_notes/365-release-notes.adoc:50]: same suggestion as above

**Review by skrthomas — COMMENTED** [review]
a couple of suggestions, otherwise LGTM!

**Comment by gaurav-nelson** [comment]
@skrthomas Please don't merge this yet, this is scheduled for release tomorrow 22 sep 2021.

**Pull request merged**